config/jobs/kubernetes-sigs/kjob/kjob-periodics-main.yaml

```diff
@@ -117,8 +252,8 @@ periodics:
             privileged: true
           resources:
             requests:
-              cpu: "7"
```

Comment:
This looks like old values still? Did we forget about it in the previous PR? Or not rebased yet?

Reply:
Yeah, not rebased. Fixed.